packages/ui/src/ui/components/CellPreviewModal/CellPreviewModal.tsx

```diff
@@ -3,23 +3,23 @@ import React from 'react';
 import {useDispatch, useSelector} from 'react-redux';
```

Comment:
React-components linked with store we put to `src/ui/containers`, and `src/ui/components` should contain components independed from store.